partcad/src/partcad/shape.py

```
@@ -59,66 +59,140 @@ def __init__(self, config):
-            b3d_solid.wrapped = shape
-            b3d_solid.relocate(b3d.Location(*self.config["offset"]))
-            shape = b3d_solid.wrapped
-        if "scale" in self.config:
-            b3d_solid = b3d.Solid.make_box(1, 1, 1)
-            b3d_solid.wrapped = shape
-            b3d_solid = b3d_solid.scale(self.config["scale"])
-            shape = b3d_solid.wrapped
-        return shape
-
-    async def get_cadquery(self):
+    async def get_wrapped(self, ctx):
```

Ruff (0.8.2)

`get_wrapped` is too complex (14 > 10)

(C901)